WORKSPACE

```diff
@@ -46,36 +54,40 @@ maven_install(
     fetch_sources = True,
 )
 
-
 # Load @vaticle_bazel_distribution_pip
-load("//pip:deps.bzl", pip_deps = "deps")
-pip_deps()
+load("//pip:deps.bzl", vaticle_bazel_distribution_pip_deps = "deps")
+vaticle_bazel_distribution_pip_deps()
+load("@vaticle_bazel_distribution_pip//:requirements.bzl", "install_deps")
+install_deps()
 
 # Load @rules_pkg
 load("//common:deps.bzl", "rules_pkg")
 rules_pkg()
-load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
+load("@rules_pkg//pkg:deps.bzl", "rules_pkg_dependencies")
 rules_pkg_dependencies()
 
 # TODO: remove this declaration once we upgrade to @io_bazel_stardoc with Bazel 5 support
 # Load @bazel_skylib
 http_archive(
     name = "bazel_skylib",
-    sha256 = "f24ab666394232f834f74d19e2ff142b0af17466ea0c69a3f4c276ee75f6efce",
+    sha256 = "66ffd9315665bfaafc96b52278f57c7e2dd09f5ede279ea6d39b2be471e7e3aa",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.4.0/bazel-skylib-1.4.0.tar.gz",
-        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.4.0/bazel-skylib-1.4.0.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.4.2/bazel-skylib-1.4.2.tar.gz",
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.4.2/bazel-skylib-1.4.2.tar.gz",
```

Comment:
Upgrade skylib. We still need to include it explicitly bceause some of the stardoc targets need to declare an explicit dependency on skydoc